sire/standard-library.md

```diff
@@ -282,22 +281,15 @@ In the above example, `one`, `two` and `three` are string keys; `1` `2` and `3`
 
 ```sire
 ;; this is a valid tab:
-= t | _MkTab [b#{I'm a bar} {I'm a string}] [b#{bar value} {string value}]
+= t | tabFromPairs [[b#{one} 1] [b#{two} 2]]
 
 ;; but the same keys/values will choke the #[] version:
-= t | #[b#{I'm a bar}={I'm a string}] [b#{bar value}={string value}]
+= t | #[b#{one}=1] [b#{2}=2]
```

Comment:
```suggestion
= t | #[b#{one}=1 b#{2}=2]
```

Reply:
I'm realizing now that neither of these parse correctly.

Reply:
...oh wow okay coffee hasn't landed yet apparently. Right, that's the point of this section - these are examples of problems.